Core/HLE/sceCtrl.h

```diff
@@ -87,3 +87,14 @@ void __CtrlPeekAnalog(int stick, float *x, float *y);
 u32 __CtrlReadLatch();
 
 void Register_sceCtrl_driver();
+
+u16 GetRightVibration();
+u16 GetLeftVibration();
+
+class SceCtrl {
```

Comment:
It's a little weird to have a class/struct (that isn't static) just to access global variables... maybe this should just be a namespace?

GetRightVibration/GetLeftVibration should be prefixed with Ctrl or else in a namespace too.  Upside/downside of namespaces is that they can be convenient, but can also quickly become unwieldy.

-[Unknown]

Reply:
Few minutes ago I didn't even know something like namespace existed, thanks :)